admin/base/views.py

```diff
@@ -5,14 +5,13 @@
 from django.utils.http import urlsafe_base64_decode
 from admin.common_auth.models import MyUser
 
-
 @login_required
 def home(request):
     context = {'user': request.user}
     return render(request, 'home.html', context)
 
 def password_reset_done(request, **kwargs):
```

Comment:
It's been a while but I think I was talking about the password_reset_done method. Interestingly, it does seem to be used in the urls file. Not sure why the silver searcher didn't find it...